### PR TITLE
 Add kenhkelly's convoluted hello world program

### PR DESCRIPTION
**Pull request by kenhkelly**
Adds my fun iteration of hello world.

### CODE REVIEW

File: scripts/hello_world_kenhkelly.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package main
```
Comment (StevenCopeland) [inline scripts/hello_world_kenhkelly.go]: Please read over the requirements for a script file. Issue #1292

Comment (kenhkelly) [reply scripts/hello_world_kenhkelly.go]: I read this and then forgot to add it! It has been added. Thanks.

**Comment by kenhkelly** [comment]
I have added the missing header. Sorry about that.

File: scripts/hello_world_kenhkelly.go (hunk 1 above)
Comment (StevenCopeland) [reply scripts/hello_world_kenhkelly.go]: :+1:

**Comment by Roshanjossey** [comment]
Hi  @kenhkelly, I'm working on a similar project to help beginners to get started with contributing to open source projects.

Do check it out https://github.com/Roshanjossey/first-contributions

**Pull request merged**